package.json

```diff
@@ -51,6 +51,7 @@
     "@fastly/http-compute-js": "1.1.5",
     "@netlify/blobs": "^8.2.0",
     "@netlify/build": "^32.2.0",
+    "@netlify/config": "^23.0.1",
```

Comment:
Added dev dependency is used only in integration tests - it's not being used otherwise